### DISCUSSION TITLE
Building iOS failed xcframework

### DISCUSSION THREAD

**Discussion by amedeopro** [codemagic-ci-cd/codemagic-docs#2799] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. push to my main branch
2. running the build on codemagic
3. build ios

### Expected results

I expect the build process and release to the stores to be finished

### Actual results

During the iOS build phase I get this error:

Archiving com.trackcast.trackCast...
Automatically signing iOS for device deployment using specified development team in Xcode project: QXD6X2R29H
Running pod install...                                             13.8s
Running Xcode build...                                          
Xcode archive done.                                          5.2s
Failed to build iOS app
Error (Xcode): There is no XCFramework found at '//development/flutter/bin/cache/artifacts/engine/ios/Flutter.xcframework'.

Encountered error while archiving for device.

Build failed :|
Failed to build for iOS

### Build id (optional)

671f3829306b672686627369

**Comment by dam-ease** [comment, marked as answer]
This has been resolved on discord.
Deleting `Flutter.xcframework` set to required in `"Build Phases" tab --> Link Binary with Libraries` solved this issue. Closing this from here.
